packages/@okta/vuepress-site/docs/reference/api/asa/service-users/index.md

```diff
@@ -0,0 +1,540 @@
+| --------- | ----------- | ------------- |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint requires an object with the following fields.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `key_id`   | string | The ID of the API key. |
+| `key_secret`   | string | The secret associated with the API key. |
+
+#### Response body
+This endpoint returns an object with the following fields and a `200` code on a successful call.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `bearer_token`   | string | The JSON Web Token used to authenticate against the ASA API. |
+| `expires_at`   | string | The time at which the token expires, formatted in accordance with RFC 3339. |
+| `team_name`   | string | The name of the Team this token is for. |
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Authorization: Bearer ${jwt}" \
+--data '{
+	"key_id": "6052868b-1b04-4a14-8288-e6496d7f2f4b",
+	"key_secret": "uF0SoVBVQP/hJmQSLUZdM2a7ArYzjD8ykzvG7n4tKaOEfSErcwMUUDWpEf4Q42/HaVKPZUfILkzy/bsQFv7WRg==",
+	"roles": null
+}' \
+https://app.scaleft.com/v1/teams/${team_name}/service_token
+```
+
+##### Response
+
+```json
+{
+	"bearer_token": "eyJhbGciOiJSUzI1NiIsImtpZCI6ImI4YTAzODA0MTM0NjctNGU5OC04ZDU2LTAxNDRlNGNkMGViMCIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0NzY4MTE1OTAsImlzcyI6InNjYWxlZnQuYXV0aC50b2tlbiIsImp0aSI6IjA0YjRhNTE4LWU5YzYtNDc1My05YWY4LThlOTAwNjVjNjX5YSIsInJvbGVzIjp7IjE0NzY4MTE1OTAiOm51bGx9LCJ0ZWFtIjoic2NhbGVmdCIsInVzZXIiOiJyb2JvdF9ydXNzZWxsIn0.pHuv06Q1-sKjHrGXUzQi-uM7AAG3K1Q6rpuxycR2Py6QHwLrto1uZmZt4wrBo6tQRCl3RjHBKGcDmEfBZ6_gFnckpFMkIXUT3sIDmOvSACthgprcXjfYh0KarEDmDnIMsEPl7FVhl4N_I0yLK9O5XSS07AvAc-7RRD8udpo7inIDTRCCEvoJ16osgL1IzoDvc7ZPDj8-xhJ_kAsKc-vJ5WYKLAlCFx_fixayM43Apg2TySNE5nSeJFCa02F4ViZleY7K2l4h_p143DzVZjWEBeKmyQVRXhbZzL6HwONQckhgp_LHuSrP_sOtVc7BrFwmZq2NZtXEOWyQWfJ4Yp0qg8NzV2LhKLc4LncpQuagf8OA8jyEeQwbu3Rq9zp0y-FodMg64qfWcPSu53HiwYC9dvgw5zhXa8zuZRcPMO_orCPVYnmO761xAfsp-P8aJJZDqpxlzKA0s-ClRsgunC5C9Xq5snIq-f4hT45u8ldBfAr6dLkDO8BdPMTzufH52bTWX3iJ1ipW1YqMefJhPMzBHLwnJ3SYWN7WTEuRyoC6ndQ60PcEzsPJYAO5MxdY4WsnYOqv3aIryuTlwW3K0dNqcvBSirgxv5X7AvidO-JQLSXXrY134BxgyLBze7FwVYwH7ZhzBdX-DwsUwQsJ7R0mRWxpNnhGu7NjkbB0-QJs",
+	"expires_at": "2016-10-18T17:26:30Z",
+	"team_name": "scaleft"
+}
+```
+### List the Service Users for a Team
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/service_users" />
+List all the Service Users for a Team
+
+This endpoint requires one of the following roles: `access_admin`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+| Parameter | Type   | Description |
```

Comment:
believe that I have covered all of these parameters with comments in other PRs to address globally.